src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -437,11 +439,11 @@ protected FeedbackResponseComment getEntity(FeedbackResponseCommentAttributes at
         if (id != null) {
             return getEntityQueryKeys(id);
         }
-
         return load()
                 .filter("courseId =", attributes.courseId)
+                .filter("feedbackResponseId =", attributes.feedbackResponseId)
```

Comment:
I have changed the way we check if comment is already present in Db. `InstructorFeedbackSubmitPageUiTest` was failing because some comments were randomly not getting added to Db. Upon investigating I found this function thought comment is already in Db as all the attributes were same including createdAt.